packages/@okta/vuepress-site/docs/guides/oin-oidc-guide/protocol-level-requirements/index.md

```diff
@@ -0,0 +1,91 @@
+  * `id_token` &mdash; the ID token contains the specific info (in a JWT format) about your user that is needed by the application
+  * Optional. `refresh_token` &mdash; this token is returned if the `offline_access` scope is granted
+* The access token is then used by your application to retrieve the protected resources and information requested at the Resource Server for the End User.
+
+The Authorization code process flow depends on Okta and your SaaS back-end systems being able to securely communicate the Client Secret.
+
+Note that your application needs to verify user identity through a user store, otherwise your app could be spoofed by fake applications in the Android or Apple app stores or an End User could view and possibly modify your code. For this reason, OIDC native, mobile, and single page application (SPA) app integrations are not acceptable for OIDC app integrations in the OIN. Your application code must be set up in a flow so that your client application talks to your SaaS back-end which then communicates with Okta.
+
+To support the potentially large numbers of Okta orgs accessing it through the OIN, an OIDC integration can't use a custom authorization server, including the `default` server.
+
+A full guide to setting up the application to use the Authorization Code Flow can be found in our Okta developer guide: Implement the Authorization Code Flow
+
+Also, if you’ve used SAML for SSO in the past, it’s important to realize that the OAuth/OIDC flow is different, because OAuth/OIDC is not just an assertion that is exchanged between Okta and your SaaS back-end, but a long-term token that can be used for callback into Okta at any point as long as the token is valid.
+
+### Scopes
+
+Your OIDC client needs to use scope values to define the access privileges that are being requested for Access Tokens. The scopes associated with Access Tokens determine what resources are available when the tokens are used to access the protected endpoints. Scopes can be used to request that specific “sets” of values be available as claim information about the End User.
+
+The only required scope that has to be declared is the `openid`. When the Authentication Request is sent to Okta, then the `openid` scope identifies the request as being an OIDC request.
+
+Other optional scopes available (these are returned from the /userinfo endpoint):
+
+* `profile` &mdash; the end user's default profile claims: `name`, `family_name`, `given_name`, `middle_name`, `nickname`, `preferred_username`, `profile`, `picture`, `website`, `gender`, `birthdate`, `zoneinfo`, `locale`, and `updated_at`
+* `email` &mdash; requests access to the email and email_verified claims
+* `address` &mdash; requests access to the address claim
+* `phone` &mdash; requests access to the phone_number and phone_number_verified claims
+* `groups` &mdash; requests access to the groups claim. This is a custom scope for Okta (see below)
```

Comment:
```suggestion
* `groups` &mdash; requests access to the `groups` claim. This is a custom scope for Okta (see below)
```

Below where?

Reply:
Removed "(see below)" as it was vague reference to the explanation that followed.